src/pages/api/register.ts

```diff
@@ -9,17 +12,14 @@ import { saleorApp } from "../../saleor-app";
 export default createAppRegisterHandler({
   apl: saleorApp.apl,
   allowedSaleorUrls: [
-    /**
-     * You may want your app to work only for certain Saleor instances.
-     *
-     * Your app can work for every Saleor that installs it, but you can
-     * limit it here
-     *
-     * By default, every url is allowed.
-     *
-     * URL should be a full graphQL address, usually starting with https:// and ending with /graphql/
-     *
-     * Alternatively pass a function
-     */
+    (url) => {
+      if (allowedUrlsPattern) {
+        const regex = new RegExp(allowedUrlsPattern);
```

Comment:
**Question:** Security concern

Will this setting be available to our end users? If yes I'm wondering if there will be the case they provide regex that will cause app to break 🤔

Reply:
In L6 you can see that this is coming from ENV that is set on the server side. 

So, OSS will leave it empty or lock into their own envs. Saleor Cloud sets this to be only Cloud